### ISSUE TITLE
Accessibility basics

### ISSUE THREAD

**Issue by murillojorge** [eoscostarica/lifebank#252]

**Comment by murillojorge** [comment]
Please refer to this document for more info: https://docs.google.com/document/d/1T-GiFWieY-h9p-4LIl1m88ar4j7EP1kujgMgU4KhCbU/edit?usp=sharing

**Comment by dloynaz** [comment]
Hello Jorge!

I was doing a first review to establish which are the most urgent changes. First of all, the Credentials Recovery component is not accessible via the tab key. I was also thinking that the SideNav should make a swipe out after clicking on any of the items in the menu. Using only the keyboard, I cannot access the page, after having selected some item from the sidebar. In addition, we could work on the items in the SideBar to give it a more dynamic focus. For example the animation that looks like the beating of a heart that most other buttons have. I also think that, from the document on web accessibility that you shared with me, it seems to me that the "Register" button should be a little more prominent in the background. It can be solved by size, color, border ... It also seems to me that the video should have subtitles for people with hearing problems.

Let me know which of these points you think I could begin to correct. I can also open an issue for each of them.

Ps: On the terms of use page one of the sections is not displaying correctly. If you like I can open an issue and I can also correct it.
